Source/Core/VideoCommon/PerformanceTracker.h

```diff
@@ -18,20 +18,9 @@ class PerformanceTracker
   static constexpr u64 MAX_DT_QUEUE_SIZE = 1UL << 12;
   static constexpr u64 MAX_QUALITY_GRAPH_SIZE = 1UL << 8;
 
-  static inline std::size_t IncrementIndex(const std::size_t index)
-  {
-    return (index + 1) & (MAX_DT_QUEUE_SIZE - 1);
-  }
-
-  static inline std::size_t DecrementIndex(const std::size_t index)
-  {
-    return (index - 1) & (MAX_DT_QUEUE_SIZE - 1);
-  }
-
-  static inline std::size_t GetDifference(const std::size_t begin, const std::size_t end)
-  {
-    return (end - begin) & (MAX_DT_QUEUE_SIZE - 1);
-  }
+  static u64 IncrementIndex(u64 i) { return (i + 1) & (MAX_DT_QUEUE_SIZE - 1); }
+  static u64 DecrementIndex(u64 i) { return (i - 1) & (MAX_DT_QUEUE_SIZE - 1); }
+  static u64 GetDifference(u64 b, u64 e) { return (e - b) & (MAX_DT_QUEUE_SIZE - 1); }
```

Comment:
I still don't understand why you changed this. `size_t` is the correct type for array indices.